packages/db/README.md

```diff
@@ -36,9 +37,17 @@ db: {
   enabled: true
 }
 ```
+
 Note: Enabling Truffle DB does not affect artifacts, but will produce a new `.db`
 directory when you compile or migrate your project.
 
+Add the following to your user truffle config (`~/.config/truffle-nodejs/config.json`) to enable saving the `.db` directory to your project directory
```

Comment:
So there's some infrastructure for doing this automatically via `truffle config set <...>`. I forget the exact syntax; @eggplantzzz or @cds-amal do you remember offhand?

In any event, we probably want to recommend that approach rather than forcing users to find this file and edit it manually - since the path differs based on OS (Mac will use `~/Library/Application\ Support/truffle-nodejs`, for instance)

Reply:
Oh fantastic. 

I wasn't aware of the convenience command. I can probably drill down - `packages/core/lib/commands/config.js` (but wouldn't mind the exact syntax).

Reply:
Unfortunately [the infrastructure isn't quite there yet](https://github.com/trufflesuite/truffle/blob/644c4dea08c2313144642b10d4de100a4c2bd613/packages/core/lib/commands/config.js#L90-L91)

Reply:
Maybe we need to do a separate PR to get this TODO addressed, and then rebase this once that's merged?

Reply:
I can jump over to tackle it since it's all connected.